docs/mdoc/index.md

```diff
@@ -53,6 +53,8 @@ object Main extends App {
 }
 ```
 
+Try also [our giter8 template](https://github.com/finch/finch.g8) to get started: `sbt new finch/finch.g8`.
```

Comment:
I didn't realize it was quite old. Perhaps merge https://github.com/finch/finch.g8/pull/2 to get it into a better shape?